test/sample-project/app.json

```diff
@@ -1,28 +0,0 @@
-{
```

Comment:
Why removing this ? What was it's purpose ?

Reply:
It was a file used to deploy on heroku automatically. Today this command disappeared.
This file was generated with an old version of Lumber in the `test/sample-project` folder.
So it's useless.

Reply:
Okey good to know